docs/egg_model_example.rst

```diff
@@ -0,0 +1,149 @@
+==========================
+The Egg model case
+==========================
+
+
+The Egg model  is a synthetic reservoir model developed by `J. D. Jansen et al., 2013 <https://rmets.onlinelibrary.wiley.com/doi/pdf/10.1002/gdj3.21>`_ [1]. We used the Egg model in the Computational Geosciences group at SINTEF Digital as a test case for developing data-driven models and testing new calibration methods for history matching.
+
+.. image:: ./_static/sintef_logo.jpg
+   :scale: 10%
```

Comment:
I suggest removing the SINTEF reference here and rather create an "About FlowNet" page where Equinor, SINTEF and TNO's logos are gathered with a short story. We could also spread a few "With support of [logos]" throught the documentation (for example the home page) and maybe also on the main page of the git Repo. @anders-kiaer What do you think about that suggestion?

```suggestion
The Egg model  is a synthetic reservoir model developed by `J. D. Jansen et al., 2013 <https://rmets.onlinelibrary.wiley.com/doi/pdf/10.1002/gdj3.21>`_ [1].
```